deb/debian/stackdriver-agent.init

```diff
@@ -179,7 +137,12 @@ d_start() {
         export https_proxy=$PROXY_URL
     fi
 
-    if ! gen_config || ! check_config; then
+    if ! check_credentials; then
+        log_failure_msg "not starting, credentials error"
```

Comment:
Let's actually fold the credentials check into the config check. More specifically, it's possible to override the credentials file in the configuration, so while today we're checking for the default one, we won't always do that, and then we'd have to refactor this code -- might as well do it now.

Reply:
Done.